lib/to_currency_select_tag.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+require 'form_options_helper'
```

Comment:
RuboCop wants a blank line between the `#frozen_string …` line and the next line.

Reply:
strange. My rubocop instance didn't complain. I've added it though and pushed 👍